pkg/controller/controller_utils.go

```diff
@@ -44,7 +44,7 @@ import (
 	"k8s.io/client-go/tools/cache"
 	"k8s.io/client-go/tools/record"
 	clientretry "k8s.io/client-go/util/retry"
-	podutil "k8s.io/kubernetes/pkg/api/v1/pod"
+	podutil "k8s.io/component-helpers/pod"
```

Comment:
Each new library has to go inside of the folders owned by a SIG.

Reply:
Added a `OWNERS` file, not sure if it's sig-node that should own this folder.